Makefile

```diff
@@ -1,11 +1,25 @@
 ESLINT=./node_modules/.bin/eslint
 NODE= NODE_OPTIONS=--max_old_space_size=8000 node
 SASSLINT=./node_modules/.bin/sass-lint -v
+SCRATCH_DOCKER_CONFIG=./node_modules/.bin/docker_config.sh
 S3CMD=s3cmd sync -P --delete-removed --add-header=Cache-Control:no-cache,public,max-age=3600
 TAP=./node_modules/.bin/tap
 WATCH= NODE_OPTIONS=--max_old_space_size=8000 ./node_modules/.bin/watch
 WEBPACK= NODE_OPTIONS=--max_old_space_size=8000 ./node_modules/.bin/webpack
 
+
+# ------------------------------------
+
+$(SCRATCH_DOCKER_CONFIG):
+	npm install scratch-docker
```

Comment:
Please remove, since it should be a dependency

Reply:
But!  Part of the point of using docker is to have the npm install happen inside the docker container, which may not happen until after you've run docker-compose up (aka make docker-up), which means `scratch-docker` won't be available as a dependency yet.